### PR TITLE
*: update DinD script & tutorial to simplify DinD setup

### PR DESCRIPTION
**Pull request by tennix**
This PR updates DinD tutorial, to simplify DinD setup the script is also updated for `helm` and `kubectl` section. This makes both `helm` client and `kubectl` prerequisites.
PTAL @gregwebs @onlymellb @lilin90

### CODE REVIEW

File: manifests/local-dind/dind-cluster-v1.10.sh [hunk 1]
```diff
@@ -1845,8 +1844,9 @@ function dind::run_tiller {
     dind::step "Deploying tiller"
     hash helm 2>/dev/null
     if [[ $? -eq 0 ]];then
+	helm_version=$(helm version -c --template '{{.Client.SemVer}}')
```
Comment (onlymellb) [inline manifests/local-dind/dind-cluster-v1.10.sh]: Indent alignment

**Comment by gregwebs** [comment]
This looks good. When I tried to run it it failed while setting up the last tikv. Probably unrelated to this change

```
2018/08/24 17:37:00.301 util.rs:385: [INFO] connect to PD endpoint: "demo-pd:2379"
2018/08/24 17:37:00.509 util.rs:385: [INFO] connect to PD endpoint: "http://demo-pd-1.demo-pd-peer.tidb.svc:2379"
2018/08/24 17:37:00.609 util.rs:385: [INFO] connect to PD endpoint: "http://demo-pd-0.demo-pd-peer.tidb.svc:2379"
2018/08/24 17:37:00.642 util.rs:443: [INFO] connect to PD leader "http://demo-pd-0.demo-pd-peer.tidb.svc:2379"
2018/08/24 17:37:00.642 util.rs:373: [INFO] All PD endpoints are consistent: ["demo-pd:2379"]
2018/08/24 17:37:00.643 tikv-server.rs:425: [INFO] connect to PD cluster 6593336833212208865
2018/08/24 17:37:00.680 mod.rs:346: [INFO] starting working thread: store address resolve worker
2018/08/24 17:37:00.825 mod.rs:425: [INFO] storage RaftKv started.
2018/08/24 17:37:00.881 server.rs:99: [INFO] listening on 0.0.0.0:20160
2018/08/24 17:37:00.885 node.rs:202: [ERROR] cluster ID mismatch: local_id 6591801254301479671 remote_id 6593336833212208865. you are trying to connect to another cluster, please reconnect to the correct PD
```

**Comment by tennix** [comment]
@gregwebs This error happens because TiKV uses old data that doesn't belong to the current TiDB cluster. Did you remove the data after you destroy the DinD cluster? `dind-cluster-v1.10.sh clean` doesn't clean up data automatically because data dir on Linux is owned by the root user, removing it requires root privilege.

**Pull request merged**